subprojects/docs/src/snippets/tutorial/count/groovy/build.gradle

```diff
@@ -1,5 +1,12 @@
+class Example {
```

Comment:
I am curious why these `Example` classes were added.

Did you intentionally add these in this PR or was this left over from testing? They do not seem to contribute anything additional to the example in `tutorial_using_tasks.adoc`.

Reply:
Removing these are they are not for this PR.